qa/tasks/ceph.py

```diff
@@ -611,15 +611,20 @@ def cluster(ctx, config):
     for remote, roles_for_host in osds.remotes.items():
         devs = teuthology.get_scratch_devices(remote)
         roles_to_devs = {}
-        if config.get('fs'):
-            log.info('fs option selected, checking for "scratch" (osd) devs')
-            log.info('found "scratch" (osd) devs: {}'.format(devs))
-            roles_to_devs = assign_devs(
-                teuthology.cluster_roles_of_type(roles_for_host, 'osd', cluster_name), devs
-            )
-            devs_to_clean[remote] = []
+        fs_val = config.get('fs')
+        assert fs_val, \
```

Comment:
fs doesn't apply to bluestore (even if configs happen to have it, it makes no sense since bluestore uses the raw block device)

Reply:
@smithfarm i concur with Josh that "fs" is not necessary when it comes bluestore. i don't think we need to check it and assert its existence.